modules/nw-anp-nodes-peer-concept.adoc

```diff
@@ -0,0 +1,232 @@
+
+.Example of ANP and BANP using `networks` peers
+[%collapsible]
+====
+[source,yaml]
+----
+apiVersion: policy.networking.k8s.io/v1alpha1
+kind: AdminNetworkPolicy
+metadata:
+  name: network-as-egress-peer
+spec:
+  priority: 70
+  subject:
+    namespaces: {}
+  egress:
+  - name: "deny-egress-to-external-dns-servers"
+    action: "Deny"
+    to:
+    - networks:
+      - 8.8.8.8/32
+      - 8.8.4.4/32
+      - 208.67.222.222/32
+    - pods:
+        namespaceSelector: {}
+        podSelector: {}
+  - name: "allow-all-egress-to-intranet"
+    action: "Allow"
+    to:
+    - networks:
+      - 172.30.0.10/32 <1>
+    - pods:
+        namespaceSelector: {}
+        podSelector: {}
+    ports:
+      - portNumber:
+          protocol: UDP
+          port: 53
+  - name: "allow-all-intra-cluster-traffic"
+    action: "Allow"
+    to:
+    - networks:
+      - 10.128.0.0/14
+      - 172.30.0.0/16 <2>
+    - pods:
+        namespaceSelector: {}
+        podSelector: {}
+  - name: "pass-all-egress-to-internet"
+    action: "Pass"
+    to:
+    - networks:
+      - 0.0.0.0/0 <3>
+---
+apiVersion: policy.networking.k8s.io/v1alpha1
+kind: BaselineAdminNetworkPolicy
+metadata:
+  name: default
+spec:
+  subject:
+    namespaces: {}
+  egress:
+  - name: "deny-all-egress-to-internet"
+    action: "Deny"
+    to:
+    - networks:
+      - 0.0.0.0/0 <3>
+---
+----
+<1> Use `networks` to specify a range of CIDR networks outside of the cluster.
+<2> Specifies the CIDR ranges for the intra-cluster traffic from your resources.
+<3> Specifies a strong `Deny` egress policy to the internet by setting `networks` values to - 0.0.0.0/0. Make sure you have an allow policy at a higher priority before setting a `Deny` to 0.0.0.0/0 because this will deny all traffic including to API servers.
+====
+
+Collectively the "Example of ANP and BANP using `networks` peers enforces the following egress policy:
+
+* All pods cannot talk to company's intranet DNS servers.
+
+* All pods can talk to rest of the company's intranet.
+
+* All pods can talk to other pods, nodes, and services.
+
+* All pods cannot talk to the internet. Combining the last ANP `Pass` rule and the strong BANP `Deny` rule a guardrail policy is created that secures traffic in the cluster.
+
+[id="combined-nodes-peer-networks-peer-anp"]
+== Using nodes peer and networks peer together
+
+Cluster administrators can combine `nodes` and `networks` peer in your ANP and BANP policies.
+
+.Example of combined `nodes` and `networks` peer
+[%collapsible]
+====
+[source,yaml]
+----
+apiVersion: policy.networking.k8s.io/v1alpha1
+kind: AdminNetworkPolicy
+metadata:
+  name: egress-peer-1 <1>
+spec:
+  egress: <2>
+  - action: "Allow"
+    name: "allow-egress"
+    to:
+    - nodes: <3>
+        matchExpressions:
+        - key: worker-group
+          operator: In
+          values:
+          - workloads # Egress traffic from nodes with label worker-group: workloads is allowed.
+    - networks:
```

Comment:
4 annotation for networks is missing?

Reply:
Sorry, I'm not sure what happened here. I mislabeled. I fixed these. I didn't think it was worth re-annotating this example too heavily given the comments so I tried to emphasize higher level things `name` priorit, etc